ChangeLog

```diff
@@ -122,6 +122,11 @@ Features
      and hmac_demo.c, which use PSA and the md/cipher interfaces side
      by side in order to illustrate how the operation is performed in PSA.
      Addresses #5208.
+   * Brought in PSA code gen driver list JSON,
```

Comment:
plz no txtspk in chglog

Reply:
Also you need to add a file to `Changelog.d`, don't edit `ChangeLog` directly (if we edit `ChangeLog` directly we get conflicts between all the pull requests that do it).

Reply:
@gilles-peskine-arm `Changelog` issue is addressed, if there is anything please let me.